README.md

```diff
@@ -99,6 +99,45 @@ from the Debug > Theme menu. There is not yet a mechanism for including
 custom themes.
 
 
+## CLI
+
+Xi-Mac includes a CLI for opening files directly from the command line.
+
+### Installing
+
+1. Build a release build of XiCli in Xcode  
+2. Execute the following commands in the project directory:
+
+```bash
+> cd Build/Release
+> cp -f XiCli /usr/local/bin/xi  
+> cp -f -R XiCliCore.framework /usr/local/bin/XiCliCore.framework
```

Comment:
Do you have any preliminary ideas on how to go about that?

Reply:
I'm not totally sure, but some rough ideas:

- we want to build a static binary; that is, we want our binary to not link to a dynamic library ( a framework, in apple land). A simple way to do this would be have the file that is now in `XiCliCore` just be part of the `XiCli` target/project. An alternative would be to have `XiCliCore` be a static library instead of a framework; I'm not sure exactly how this works in swift packages, but it _should_ just be a flag you set in some config file.

- we want to build that static binary as part of the normal build process. This means, I think, making sure that that target gets built as part of the default build scheme; and then we will want a 'build phase' where we copy that target into the package. This should be similar to how we bundle xi-core; as a build step in xcode we run a custom script that compiles the `xi-core`, and then we copy this file into our package so that we can launch it at runtime.

I hope that's enough to point you in the right direction, if you're stuck or confused let me know and I'll try to clarify.

Reply:
I finished switching it to static!